src/components/Coldcard/ColdcardExtendedPublicKeyImporter.jsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { COLDCARD } from "unchained-wallets";
+import { FormGroup, FormHelperText } from "@material-ui/core";
+import { MAINNET, P2SH } from "unchained-bitcoin";
+import { ColdcardJSONReader } from ".";
+import IndirectExtendedPublicKeyImporter from "../Wallet/IndirectExtendedPublicKeyImporter";
+
+class ColdcardExtendedPublicKeyImporter extends React.Component {
+  constructor(props) {
+    super(props);
+    const coldcardBIP32Path = this.getColdcardBip32Path();
+    this.state = {
+      COLDCARD_MULTISIG_BIP32_PATH: coldcardBIP32Path,
+    };
+  }
+
+  componentDidMount = () => {
+    const { extendedPublicKeyImporter, validateAndSetBIP32Path } = this.props;
+    const { COLDCARD_MULTISIG_BIP32_PATH } = this.state;
+    if (extendedPublicKeyImporter.method === COLDCARD) {
+      validateAndSetBIP32Path(
+        COLDCARD_MULTISIG_BIP32_PATH,
+        () => {},
```

Comment:
Doesn't need to be something that we delay a merge over, but the fact that there are so many empty args that need to be passed in makes me feel like this function signature is not as clean or user friendly as it could be.

Reply:
Agree